pinot-tools/pom.xml

```diff
@@ -110,6 +110,10 @@
       <artifactId>testng</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I don't think you need to add this dependency explicitly as it is already included

Reply:
That's good to have. Maven uses BFS to get the list of classpath (https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Transitive_Dependencies). 
The issue that we encounter in some of the machines is that tyrus dependency comes before the grizzly dep. Putting grizzly dependency before tyrus can make sure the class grizzly will be loaded first.